examples/commandr/convert_checkpoint.py

```diff
@@ -8,7 +9,7 @@
 from tensorrt_llm._utils import release_gc
 from tensorrt_llm.logger import logger
 from tensorrt_llm.mapping import Mapping
-from tensorrt_llm.models import CohereForCausalLM
+from tensorrt_llm.models import CohereForCausalLM, Cohere2ForCausalLM
```

Comment:
Could you please update `examples/commandr/readme.md` to show how to run a Cohere2 model?

Reply:
Please also mention in a note that we extract the `max_attention_window` from the HF config